cypress/integration/1217-my_profile_page/1724-non-customer_views_my_profile/non_customer_views_profile.js

```diff
@@ -0,0 +1,22 @@
+import { Given, When, Then, And } from 'cypress-cucumber-preprocessor/steps';
+import { USER_NON_CUSTOMER } from '../../../support/constants';
+
+Given('that a User is logged in with Feide', () => {
+  cy.login(USER_NON_CUSTOMER);
+});
+And('their Institution is not a Customer of NVA', () => {});
+When('they navigate to My Profile', () => {
+  cy.get('[data-testid=menu]').click({ force: true });
+  cy.get('[data-testid=menu-user-profile-button]').click({ force: true });
+});
+Then('they see that they have no roles', () => {
+  cy.get('[data-testid^=user-role]').should('not.exist');
+});
+And('they see an information text explaining why they have no roles', () => {
+  cy.get('[data-testid=no-roles-text]').should('be.visible');
+});
+And('they see their Institution name', () => {
+  cy.get('[data-testid=institution-presentation]').within(($presentation) => {
+    cy.wrap($presentation).contains('Norwegian University of Science and Technology').should('be.visible');

```

Comment:
kunne kanskje brukt en konstant for institusjon her også. antar dette blir knyttet til brukeren en annen plass..?